app/main.dev.ts

```diff
@@ -29,8 +29,10 @@ if (process.env.NODE_ENV === 'production') {
 }
 
 if (process.env.NODE_ENV === 'development' || process.env.DEBUG_PROD === 'true') {
-  require('electron-debug')()
+  require('electron-debug')();
```

Comment:
There seem to be recurring style errors, can you fix them? Travis builds should be green. When setting up the boilerplate, I had all the lint errors reported by webstorm. CLI `yarn lint` also works, if that suits you better.